### PR TITLE
Fix zero amount cash transactions

### LINKED ISSUE #2193: Do not allow cash transactions for zero amount
## Is your feature request related to a problem? Please describe.

Cash transactions can currently be made for an amount of `0.00`.

## Describe the solution you'd like

Do not allow a cash in/cash out to be confirmed for `0.00`.

## Implementation

Update `isValidTransaction` conditional to require amounts larger than zero.

## Describe alternatives you've considered

N/A.

## Additional context

N/A.

### PR DESCRIPTION
**Pull request by wlthomson**
Fixes #2193.

## Change summary

Disable `Confirm` button for transactions with zero amounts.

## Testing

Steps to reproduce or otherwise test the changes of this PR:

- [ ] Enter an amount of `0.00` for a new cash transaction. The confirm button should be greyed out and unclickable.

### Related areas to think about

N/A.

### CODE REVIEW

File: src/widgets/modals/CashTransactionModal.js [hunk 1]
```diff
@@ -48,12 +48,10 @@ export const CashTransactionModal = ({ onConfirm }) => {
...
   );
 
-  const isValidTransaction = useMemo(() => !!name && !!amount && (isCashIn || !!reason), [
-    name,
-    amount,
-    reason,
-    isCashIn,
-  ]);
+  const isValidTransaction = useMemo(
+    () => !!name && !!amount && amount > 0 && (isCashIn || !!reason),
```
Comment (joshxg) [inline src/widgets/modals/CashTransactionModal.js]: if amount was `0` then `!!amount == false` and this expression is the same as before, no?

Comment (wlthomson) [reply src/widgets/modals/CashTransactionModal.js]: It explicitly coerces the `amount` to an `int`. Not a nice solution, but under time constraints does the job.

Comment (wlthomson) [reply src/widgets/modals/CashTransactionModal.js]: It explicitly coerces the `amount` to an `int`. Not a nice solution, but under time constraints does the job.

Comment (wlthomson) [reply src/widgets/modals/CashTransactionModal.js]: 24ccda1 (temporary fix, needs a refactor asap).

**Pull request merged**